### ISSUE TITLE
pseudoalignments.tsv error

### ISSUE THREAD

**Issue by Bondada20** [RabadanLab/arcasHLA#134]
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/arcas_e8c94069-a619-4cc1-b53c-789445ed2377/pseudoalignments.tsv'

**Comment by kalanir** [comment]
I'm also having this issue when running arcasHLA genotype

**Comment by IntelligentRock** [comment]
I have the same issue, also no such file or directory: pseudoaligments.tsv. Running arcasHLA genotype on fastq.gz files.

**Comment by ShihanL** [comment]
Looks like it's an issue with kallisto versions, must be 0.44.0. Newer versions appear to cause this bug

**Comment by IntelligentRock** [comment]
> Looks like it's an issue with kallisto versions, must be 0.44.0. Newer versions appear to cause this bug

This doens't fix it for me. I still get the same error on 0.44.0. Did you change anything else? The actual file its looking for, pseudocount.tsv, also doesn't exist in the folder, only trim.reads.txt.

**Comment by asaksager** [comment]
I also have this problem, and I cannot use kallisto 0.44.0 (0.51.0 instead) as it is not avaible.
Did you find a solution? 
@IntelligentRock @ShihanL @kalanir @andreas-wilm

**Comment by IntelligentRock** [comment]
> I also have this problem, and I cannot use kallisto 0.44.0 (0.51.0 instead) as it is not avaible. Did you find a solution? [@IntelligentRock](https://github.com/IntelligentRock) [@ShihanL](https://github.com/ShihanL) [@kalanir](https://github.com/kalanir) [@andreas-wilm](https://github.com/andreas-wilm)

No. Even with Kallisto 0.44.0 it won't work, I also encountered a lot of other issues with arcas, so I decided to use nf-core/hlatyping. I recommend trying it out to see if it meets your requirements.

**Comment by BERICP01** [comment]
Same issue!

**Comment by vivianschu** [comment]
I also have this same issue!

**Comment by Ryan-3** [comment]
I think this just happens if Kallisto fails in general. For me, it was because I missed arcasHLA genotype also required --single to indicate single-end reads. That error was in the log. Once I fixed that, it worked.

**Comment by TrofimovAssya** [comment]
Downgrading kallisto to 0.43.0 + adding --single for single-end reads fixed this issue for me
